lib/mp4/probe.js

```diff
@@ -150,7 +150,7 @@ startTime = function(timescale, fragment) {
                       tfhd[6] <<  8 |
                       tfhd[7]);
       // assume a 90kHz clock if no timescale was specified
-      scale = timescale[id] || 90e3;
+      scale = (typeof timescale === 'object' ? timescale[id] : timescale) || 90e3;
```

Comment:
are there times where timescale isn't an object?

Reply:
in the new `getTracks` function each track returns its own timescale

Reply:
Personally, I like inputs to be of a consistent type since it's easier to reason about. That said, this is fine technically.

Reply:
actually it seems like we want it to always be an object, and I just realized why. We are getting the start time via a fragment, which could have any of the track ids from the init segment.

Reply:
Made a new helper, rather than changing the type of timescale.